.mailmap

```diff
@@ -174,7 +174,6 @@ Steve Laing <steve.laing@digital.cabinet-office.gov.uk>
 Steve Laing <steve.laing@digital.cabinet-office.gov.uk> <developer@laingsolutions.com>
 Steve Laing <steve.laing@digital.cabinet-office.gov.uk> <steve.laing@gmail.com>
 Stuart Gale <stuart.gale@digital.cabinet-office.gov.uk> <bishboria@gmail.com>
-suthagarht <suthagar.kathirkamathamby@digital.cabinet-office.gov.uk> <suthagarht@gmail.com>
```

Comment:
The mailmap relates to looking at commits. As we're not removing any commits, I think the mailmap entry should be kept.

Reply:
👍